File: android/src/androidTest/java/org/ligi/gobandroidhd/uitest/TheGoTerminologyActivity.java
package org.ligi.gobandroidhd.uitest;

import android.content.Intent;
import android.net.Uri;
import android.test.suitebuilder.annotation.MediumTest;

import com.squareup.spoon.Spoon;

import org.ligi.gobandroid_hd.ui.go_terminology.GoTerminologyViewActivity;
import org.ligi.gobandroidhd.base.BaseIntegration;

public class TheGoTerminologyActivity extends BaseIntegration<GoTerminologyViewActivity> {

    public TheGoTerminologyActivity() {
        super(GoTerminologyViewActivity.class);
    }

    @MediumTest
    public void testThatGoBoardIsThere() {
        final Intent intent = new Intent();
        intent.setData(Uri.parse("goterm://miai"));
        setActivityIntent(intent);
        final GoTerminologyViewActivity activity = getActivity();

        Spoon.screenshot(activity, "go_terms");
    }


}

File: android/src/androidTest/java/org/ligi/gobandroid_hd/uitest/TheGoTerminologyActivity.kt
package org.ligi.gobandroid_hd.uitest

import android.content.Intent
import android.net.Uri
import org.junit.Rule
import org.junit.Test
import org.ligi.gobandroid_hd.ui.go_terminology.GoTerminologyViewActivity
import org.ligi.trulesk.TruleskActivityRule

class TheGoTerminologyActivity {

    @get:Rule
    val rule = TruleskActivityRule(GoTerminologyViewActivity::class.java, false)

    @Test
    fun testThatGoBoardIsThere() {

        val intent = Intent()
        intent.data = Uri.parse("goterm://miai")

        rule.launchActivity(intent)

        rule.screenShot("go_terms")
    }

}
